templates/scenarios/js/sso-tab/teamsfx/app.yml

```diff
@@ -34,7 +34,7 @@ deploy:
   - uses: azureStorage/deploy # Deploy bits to Azure Storage Static Website
     with:
       workingDirectory: .
-      distributionPath: ./build # Deploy base folder
+      distributionPath: ./build # Deploy base folder, please ensure you ignore manifest file for AAD app and Teams App in this folder in the ignoreFile
```

Comment:
. This folder includes manifest files for AAD app and Teams app that should be ignored using the ignoreFile.

Reply:
Thanks, updated.